src/foundry/foundry.js/applications/formApplications/userConfig.d.ts

```diff
@@ -2,15 +2,13 @@ import { ConfiguredDocumentClass } from '../../../../types/helperTypes';
 
 declare global {
   /**
-   * The User Configuration application provides a form used to allow the current client to edit preferences and
-   * configurations about a User entity (typically their own).
+   * The Application responsible for configuring a single User document.
    * @typeParam P - the type of the options object
    */
-  class UserConfig<P extends DocumentSheet.Options = DocumentSheet.Options> extends DocumentSheet<
-    P,
-    Data,
-    InstanceType<ConfiguredDocumentClass<typeof User>>
-  > {
+  class UserConfig<
+    P extends DocumentSheet.Options = Options,
```

Comment:
Please rename this generic parameter to `Options` for consistency. Put `Options` into the `UserConfig` namespace to avoid a conflict.